### ISSUE TITLE
Regarding the Sth-com experimental setting.

### ISSUE THREAD

**Issue by baoqianyue** [RongchangLi/ZSCAR_C2C#3]
Thank you very much for open-sourcing such excellent work and code. In your paper 3.1 The Proposed ZS-CAR Task, you mentioned, "We follow the generalised zero-shot learning setting [6,50], i.e., both seen and unseen action categories appear in the validation/test set:". Since I know little about zero-shot learning, from the data in Table 2 of your experiment, I guess that the verb column is still the result of general action recognition classification, while the seen and unseen columns are used to evaluate the performance of zero-shot learning. Is my understanding correct? I hope you can enlighten me.

**Comment by RongchangLi** [comment]
> Thank you very much for open-sourcing such excellent work and code. In your paper 3.1 The Proposed ZS-CAR Task, you mentioned, "We follow the generalised zero-shot learning setting [6,50], i.e., both seen and unseen action categories appear in the validation/test set:". Since I know little about zero-shot learning, from the data in Table 2 of your experiment, I guess that the verb column is still the result of general action recognition classification, while the seen and unseen columns are used to evaluate the performance of zero-shot learning. Is my understanding correct? I hope you can enlighten me.

Exactly! Besides, recognizing verbs is more difficult than previous action recognition setting. Though the testing verb categories are included in the training set, the combined objects of a specific verb may be 'unseen' for this verb. The unseen and seen metrics are the best accuracy of the verb-object compositions.

**Comment by baoqianyue** [comment]
Thanks for your reply！
